src/components/cc-logs-controller/cc-logs-controller.stories.js

```diff
@@ -0,0 +1,59 @@
+import './cc-logs-controller.js';
+import { makeStory } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+import { createFakeLogs, CUSTOM_METADATA_RENDERERS } from '../cc-logs/fake-logs.js';
+
+export default {
+  title: '🚧 Beta/🛠 Logs/<cc-logs-controller-beta>',
+  component: 'cc-logs-controller-beta',
+};
+
+const conf = {
+  component: 'cc-logs-controller-beta',
+  beta: true,
+  // language=CSS
+  css: `
+    cc-logs-controller-beta {
+      border: 1px solid #ddd;
+      border-radius: 0.2em;
+      padding: 0.5em;
+      height: 800px;
+    }
+  `,
+};
+
+const metadataDisplay = {
+  ip: {
+    label: 'Afficher l\'adresse IP',
+    hidden: true,
+  },
+  level: {
+    label: 'Afficher le niveau de log',
+    hidden: false,
+  },
+};
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      follow: true,
+      metadataDisplay: metadataDisplay,
+      metadataRenderers: CUSTOM_METADATA_RENDERERS,
+      logs: createFakeLogs(100, { longMessage: true, ansi: true }),
+    },
+  ],
+});
+
+export const withoutMetadataDisplay = makeStory(conf, {
```

Comment:
**question**: I am not sure to understand why do we need the case without metadata display ?

Reply:
This is to show that it is possible to not let the user change the metadata display, and that in that case, the whole section disappears from the UI.